src/pages/office/office.ts

```diff
@@ -196,10 +199,12 @@ export class OfficePage extends BasePageComponent {
         select(getAdditionalInformation),
       ),
       showMeQuestion$: currentTest$.pipe(
+        select(getTestData),
```

Comment:
As we're piping `currentTest$`, why do we need to do `getTestData` again?

Reply:
Good point. Weird that it works fine..
Will change it.

Reply:
No, you need the `getTestData` selector. The vehicle checks are part of the `testData`

Reply:
Ah yes, I understand now, `testData` is the key within the individual test, fair enough. 😄